images/Dockerfile.kcp-front-proxy

```diff
@@ -0,0 +1,6 @@
+# Use distroless as minimal base image to package the manager binary
+# Refer to https://github.com/GoogleContainerTools/distroless for more details
+FROM gcr.io/distroless/static:debug
+WORKDIR /
+COPY --from=kcp-build workspace/bin/kcp-front-proxy /
+ENTRYPOINT ["/kcp-front-proxy"]
```

Comment:
These Dockerfile all look very similar, could it be standardized in a way that we pass the names of the binary to build and the path as an argument?

Reply:
Yes I thought the same, we could but I wasn't completely sure if we'll end up with any binary-specific additions, such as those already added for the workdir etc to the kcp image - if not we could indeed use a single file with a per-binary build arg.